CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# How to Contribute to BotKube
+
+We'd love your help!
+
+BotKube is [MIT Licensed](LICENSE) and accepts contributions via GitHub
+pull requests. This document outlines some of the conventions on development
+workflow, commit message formatting, contact points and other resources to make
+it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
+
+## Compile BotKube from source code
+Make sure you have `go` compiler installed. BotKube uses
+[`dep`](https://github.com/golang/dep) to manage dependencies. You
```

Comment:
Probably break  down this section into,
Pre-requisites/Requirements
Build Steps